In [1]:
#DEPENDENCIES & INITIALISATIONS
from IPython.display import HTML #For embedding videos
import math #Basic math functionality
import numpy as np #Numpy for working with arrays
import matplotlib.pyplot as plt #Plotting functionality 
#HTML('<center><iframe width="560" height="315" src="#" frameborder="0" allowfullscreen></iframe></center>')
print("- Dependencies imported 👍")
print("- Remember to select 'Kernel-> Restart & Run All' on first load to show embedded Youtube videos 🎬")
print("- Author: Dr SPCarroll, Sept 2020, Rev 0")

- Dependencies imported 👍
- Remember to select 'Kernel-> Restart & Run All' on first load to show embedded Youtube videos 🎬
- Author: Dr SPCarroll, Sept 2020, Rev 0


# Week 8: Energy-based Analysis Methods
Reference: Chapter 15 of Engineering Mechanics - Statics and Dynamics Principles, Bedford A. & Fowler, W., 2003

## Question 1

The $2000 \:N$ drag racer starts from rest and travels a $250\:m$ course. It completes the course in $4.524$ seconds and crosses the finish line traveling at $325.77\: km/h$. 

- (a) How much work is done on the car as it travels the course? 
- (b) Assume that the horizontal force exerted on the car is constant and use the principle of work and energy to determine it.
- (c) Determine the maximum power transferred to the car as it travels the $250\:m$ course.
- (b) Determine the average power transferred to the car.

**Solution 1**

(a) We know that the work done on an object is equal to its change in kinetic energy. Therefore knowing the velocity magnitudes we can readily determine the work done using,

\begin{equation*}
U_{12} = \frac{1}{2}mv_2^2 - \frac{1}{2}mv_1^2
\end{equation*}

In [2]:
v1 = 0 #(m/s) Initial velocity
v2_kph = 325.77 #(km/h) Final velocity
v2 = v2_kph*1000/3600 #(m/s) Final velocity
w = 2000 #(N)
m = w/9.81 #(kg)
work = 0.5*m*v2**2 - 0.5*m*v1**2 #(Nm)
print('1(a). The work done on the drag racer is {one} Nm'.format(one=round(work)))

1(a). The work done on the drag racer is 834734 Nm


(b) We know that the work done is equal to the force multiplied by the distance through which it travels, therefore,

\begin{equation*}
\text{Force} = \frac{\text{Work}}{\text{distance}}
\end{equation*}

In [3]:
d = 250 #(m) Distance travelled
F = work/d #(N)
print('1(b). The constant force applied to the drag racer is {one} N'.format(one=round(F)))

1(b). The constant force applied to the drag racer is 3339 N


(c) Power is the product of force and velocity. Therefore the maximum power is achieved when the velocity is a maximum. 

In [4]:
P_max = F*v2 #(Nm/s) Maximum power
print('1(c). The maximum power is {one} Nm/s'.format(one=round(P_max)))

1(c). The maximum power is 302146 Nm/s


(d) The average power is obtained by dividing the work done by the time taken or duration,

\begin{equation*}
P_{\text{avg}} = \frac{U_{12}}{t_2-t_1}
\end{equation*}

In [5]:
t = 4.524 #(s)
P_avg = work/t
print('1(d). The average power transferred to the drag racer is {one} Nm/s'.format(one=round(P_avg)))

1(d). The average power transferred to the drag racer is 184512 Nm/s


## Question 2

As a $32,000 \:N$ aircraft takes off, the tangential component of force exerted on it by its engines is $\sum F_t = 45000 \:N$. Neglecting other forces on the aircraft, use the principle of work and energy to determine 

- (a) how much runway is required for its velocity to reach $200\:km/h$.
- (b) the average power transferred to the aircraft.

Assume the aircraft undergoes constant acceleration between $v=0\:km/h$ and $v=200\:km/h$. 

**Variable force**

Now consider a related scenario; before travelling down the runway, the aircraft is at rest at position $s=0$. The total tangential force exerted on it by its engines and aerodynamic drag (in Newtons) is given as a function of its position $s$ by,

\begin{equation*}
\sum F_t = 4500 - 5.2\:s
\end{equation*}

- (c) Use the principle of work and energy to determine how fast the airplane is traveling when its position is $s = 950\:m$.

**Solution 2**

(a) By equating the work done to the change in kinetic energy we can solve for the distance through which the force travels,

\begin{equation*}
\sum F_t\:d = \frac{1}{2}mv_2^2 - \frac{1}{2}mv_1^2
\end{equation*}

In [6]:
F = 45000 #N
w = 32000 #(N)
m=w/9.81 #(kg)
v_kph = 200 #(km/h)
v = v_kph*1000/3600 #(m/s)
d = (1/F)*(0.5*m*v**2) #(m)
print('2(a). The distance required to reach 200 km/h is {one} m'.format(one=round(d,1)))

2(a). The distance required to reach 200 km/h is 111.9 m


(b) The average power is obtained by dividing the work done by the time taken. We can work out the work done by evaluating the change in kinetic energy,

\begin{equation*}
U_{12} = \frac{1}{2}mv_2^2 - \frac{1}{2}mv_1^2
\end{equation*}

noting that the initial velocity is zero. 

In [7]:
work = 0.5*m*v**2
print('The work done is {one} Nm'.format(one=round(work)))

The work done is 5033916 Nm


Next we need to determine the time taken to reach a velocity of $200 km/h$. Assuming constant acceleration, we know,

\begin{equation*}
v_2 = v_1 + a_0\:t
\end{equation*}

\begin{equation*}
v_2 = a_0\:t
\end{equation*}

Therefore,

\begin{equation*}
t = \frac{v_2}{a_0}
\end{equation*}

To determine the acceleration we can use Newton's second law,

\begin{equation*}
a_0 = \frac{F}{m}
\end{equation*}

In [8]:
a_0 = F/m #(m/s^2)
t = v/a_0 #(s)
P_avg = work/t #(Nm/s)
print('2(b). The average power transferred is {one} Nm/s'.format(one=round(P_avg)))

2(b). The average power transferred is 1250000 Nm/s


(c) Now that the force is a function of position $s$ so we must integrate to obtain the total work done. 

\begin{equation*}
U_{12} = \int_0^{950}(45000-5.2\:s)\:\mathrm{d}s
\end{equation*} 

\begin{equation*}
U_{12} = 45000\:s - \frac{5.2s^2}{2} \biggr\rvert_0^{950}
\end{equation*}

In [9]:
s = 950 #(m)
work = 45000*s - 5.2*s**2/2
print('The work done is {one} Nm'.format(one=round(work)))

The work done is 40403500 Nm


This can now be equated to the change in kinetic energy to yield the velocity $v_2$ when $s_2 = 950\:m$. 

In [16]:
v = math.sqrt(work/(0.5*m)) #(m)
print('2(c). The velocity of the aircraft after it has travelled 950m is {one} m/s'.format(one=round(v)))

2(c). The velocity of the aircraft after it has travelled 950m is 4495 m/s


## Question 3

A stunt driver wants to drive a car through a circular loop of radius $R = 5 \:m$

- Determine the minimum required velocity at the top of the loop such that the car stays in contact with the track.

- Determine the minimum velocity $v_0$ at which the car can enter the loop and coast through without losing contact with the track. 

**Hint:**
- Model the car's motion using components normal and tangential to the circular path.

![title](img/T2W8/img4.jpg)
<center>Car coasting around a loop with an entry velocity $v_0$.</center>

**Solution 3**

We start this solution by considering a free-body diagram of the car when it is inverted at the top of the loop.

![title](img/T2W8/img5.jpg)
<center>Free-body diagram of car centre of mass at the top of the loop.</center>

We can see the self-weight of the mass, $mg$ and also the reaction force $N$ imposed on the mass by the loop. We note that at the point the car looses contact with the loop, $N\rightarrow 0$. Also indicated are the tangential and normal components of the acceleration. 

We are trying to analyse the case when $N\rightarrow 0$, as such we are concerned with forces that are normal to the circular path. Evaluating the sum of the forces in the normal direction, 

\begin{equation*}
mg + N = 0
\end{equation*}

We also know according to Newton's second law, $\sum F_n = m\times a_n$, i.e. the sum of the forces in the normal direction must equal the mass times the acceleration in the normal direction and since at the point contact is lost $N=0$, we have

\begin{align*}
m\:g &= m\:a_n\\
g &=a_n
\end{align*}

We know that the normal component of acceleration for circular motion is given by (week 5 equation 47),

\begin{equation*}
a_n = \frac{v^2}{R}
\end{equation*}

Therefore solving for $v$,

\begin{equation*}
v = \sqrt{R\:g}
\end{equation*}

In [11]:
R = 5 #(m)
v = math.sqrt(R*9.81)
print('3(a). The minimum required velocity at the top of the loop is {one} m/s'.format(one=round(v,1)))

3(a). The minimum required velocity at the top of the loop is 7.0 m/s


(b) The entry velocity can be determined by equating the work done in elevating the car $10\:m$ with the change in kinetic energy. Considering the work done first,

\begin{equation*}
\text{work} = -mg(y_2-y_1)
\end{equation*}

where the negative sign is due to the fact that negative work is done as the force points in the opposite direction to the distance travelled. 

The change in kinetic energy from the point of entry to the top of the loop is,

\begin{equation*}
U_{12} = \frac{1}{2}m v_{\text{top}} - \frac{1}{2}mv_0
\end{equation*}

Equating work with change in kinetic energy and solving for $v_0$,

In [12]:
y1 = 0 #(m)
y2 = 2*R #(m)
v0 = math.sqrt(9.81*(y2-y1)*2+v**2)
print('3(b). The required entry velocity is {one} m/s'.format(one=round(v0,2)))

3(b). The required entry velocity is 15.66 m/s


## Question 4

Consider the steel collar capable of sliding along vertical shaft shown below. The spring attached to the collar has a spring stiffness $k=100N/m$ and the collar weighs $4\:kg$. At position $1$ the spring is unstretched. 

- (a) If the collar is allowed to fall under its own weight from position 1, how far will it fall?

- (b) Now imagine a new scenario; the collar against starts at position 1 but has an initial downward velocity of $v_0=1\:m/s$. What is the maximum downward velocity the collar attains as it falls?

![title](img/T2W8/img6.jpg)
<center>Question 4.</center>

**Solution 4**

(a) We can start by considering the work done on the system. The only forces at play are  the self-weight of the collar and spring force. If we denote the distance travelled as $s$, the work done is,

\begin{equation*}
\text{work} = mg\:s - \frac{1}{2}ks^2
\end{equation*}

The work done by the spring force is negative because the direction of the spring force is opposite to the direction of motion.

Next we consider the change in kinetic energy between the initial position and the final position. Since the initial and final velocity are both zero, the change in kinetic energy is also zero. Therefore we have,

\begin{align*}
mg\:s- \frac{1}{2}ks^2 &= 0\\
s\left(mg-\frac{1}{2}ks\right) &= 0
\end{align*}

Since $s \neq 0$,

\begin{equation*}
mg-\frac{1}{2}ks = 0
\end{equation*}

Therefore solving for $s$,

In [13]:
m = 4 #(kg)
k = 100 #(N/m)
s = (2*m*9.81)/k #(m)
print('4(a). The distance travelled is {one} m'.format(one=round(s,3)))

4(a). The distance travelled is 0.785 m


<hr>

**Aside**

We can take an aside here to explore the spring behaviour before moving on; we saw in the notes this week that the potential energy in a spring is,

\begin{equation*}
V = \frac{1}{2}ks^2
\end{equation*}

And in the example above we stated that the work done in extending a spring was,

\begin{equation*}
W_{\text{spring}} = -\frac{1}{2}ks^2
\end{equation*}

We can derive this expression for work done by considering the force-displacement graph for a linear spring.

![title](img/T2W8/img7.jpg)
<center>Force-displacement graph for a linear spring.</center>

We know that the work done can be obtained from the area beneath the force-displacement graph. So in the case of a spring that extends from position $S_1$ to position $S_2$ as shown in the graph above, the work done is equal to the shaded area,

\begin{equation*}
W_{\text{spring}} = - \left[kS_1(S_2-S_1) + \frac{1}{2}(kS_2-kS_1)(S_2-S_1)\right]
\end{equation*}

Where the negative indicates that the force in the spring acts to oppose the extension (or compression) of the spring. After simplifying, this yields

\begin{equation*}
W_{\text{spring}} - \frac{1}{2}k\left(S_2^2 - S_1^2\right) = -\frac{1}{2}ks^2
\end{equation*}

<hr>

(b) We can start by equating the work done to the change in kinetic energy between positions $1$ and $2$,

\begin{equation*}
\frac{1}{2}mv_2^2 - \frac{1}{2}mv_1^2 = -\frac{ks^2}{2} + mgs
\end{equation*}

The maximum velocity is achieved when the slope of the velocity versus displacement graph is zero. 

![title](img/T2W8/img8.jpg)
<center>Sketch of velocity-displacement graph.</center>

Therefore, we need to determine $\mathrm{d}v_2/\mathrm{d}s$, let it equal to zero and solve for $s$, the position at which maximum velocity is reached. We can then substitute this back into the equation above to determine the corresponding velocity. We have used this approach to finding local maxima many time during this module. 

Rearranging our equation for $v_2^2$,

\begin{equation*}
v_2^2 = -\frac{k}{m}s^2 + 2gs + v_1^2
\end{equation*}

Differentiating both sides with respect to $s$,

\begin{equation*}
\frac{\mathrm{d}}{\mathrm{d}s}(v_2 . v_2) = \frac{\mathrm{d}}{\mathrm{d}s}(-\frac{k}{m}s^2 + 2gs + v_1^2)
\end{equation*}

\begin{equation*}
v_2\frac{\mathrm{d}v_2}{\mathrm{d}s} = -\frac{2k}{m}s + 2g
\end{equation*}

Setting $\mathrm{d}v_2/\mathrm{d}s = 0$,

\begin{equation*}
-\frac{2k}{m}s + 2g = 0
\end{equation*}

and solving for $s$,

\begin{equation*}
s = \frac{g\:m}{k}
\end{equation*}

In [14]:
s = 9.81*m/k #(m)
print('The spring position when it reaches maximum velocity is {one} m'.format(one=round(s,3)))

The spring position when it reaches maximum velocity is 0.392 m


We can now substitute this value back into our original equation relating work to change in kinetic energy and solve for $v_2$ when $s=0.392$,

In [15]:
v_1 = 1 #(m/s)
v_2 = math.sqrt(-(k/m)*(s**2) + 2*9.81*s + v_1**2)
print('4(b). The maximum velocity achieved is {one} m/s'.format(one=round(v_2,2)))

4(b). The maximum velocity achieved is 2.2 m/s


## Answers

1. $834734\:Nm$, $3339\:N$, $302146\:Nm/s$, $184512\:Nm/s$
2. $111.9 m$, $1250000\:Nm/s$, $157\:m/s$
3. $7.0\:m/s$, $15.66 m/s$
4. $0.785 m$, $2.2 m/s$